install

```diff
@@ -12,36 +12,34 @@ command -v git > /dev/null || {
 	exit 2;
 }
 
+# Get location where bro will be installed
+read -p "BRO_STATION? ($HOME/.bro):" BRO_STATION
+if [ "$BRO_STATION" = ""  ]; then
+	BRO_STATION="$HOME/.bro"
+fi
```

Comment:
In my case, the first installation attempt was failed, created orphan environment varialbes and paths. Both way checks might be rquired, in case of installation.